### ISSUE TITLE
entities arriving at UI with zero shape count

### ISSUE THREAD

**Issue by jedirv** [SCAII/SCAII#131]
When Evan and I switched to using the attack-v4 files, we started getting some issues in the UI which might be side effects of changes to support the attack-v4 files.

One issue is that we are getting some entities that have empty shape lists. Not sure of the circumstances, but we find that if we just screen those out, the game flow appears fine.

Also, jumping back and forth where the jump target is between decision points would reveal that entities   relevant to the prior DP would survive into the next DP.  I found a bug on the UI that was causing this where we needed to clear out the masterEntityList when we jump, because the jumping process causes info from the relevant keyframe to come in, but bypasses the step where the viz packet has delete signals on the entities.  However, this wasn't a problem before we were using that v4 vintage file.

Long story short - we've worked around the issues, but might be worth checking the backend to verify that its not indicative of other issues.

**Comment by LinearZoetrope** [comment]
I'm probably accidentally sending the sensors, which are no-shape entities that aren't meant to be rendered. I don't think the backend *ever* sent delete messages on jumps.
